generators/server/templates/src/test/java/package/web/rest/PublicUserResourceIT.java.ejs

```diff
@@ -180,6 +184,8 @@ class PublicUserResourceIT <% if (databaseType === 'cassandra') { %>extends Abst
             .andExpect(status().isOk())
             .andExpect(content().contentType(MediaType.APPLICATION_JSON_VALUE))
             .andExpect(jsonPath("$.[*].login").value(hasItem(DEFAULT_LOGIN)))
+            .andExpect(jsonPath("$.[*].firstName").value(hasItem(DEFAULT_FIRSTNAME)))
+            .andExpect(jsonPath("$.[*].lastName").value(hasItem(DEFAULT_LASTNAME)))            
```

Comment:
```suggestion
<%_ for (field of _.sortedUniqBy(user.otherRelationships.map(relationship => relationship.relatedField), field => field.fieldName)) { _%>
            .andExpect(jsonPath("$.[*].<%= field.fieldName %>").value(hasItem(DEFAULT_<%= field.fieldName.toUpperCase() %>)))
<%_ } _%>
```

Reply:
this code this code's making IT crash, by generate wrong constatns, for firstName should be **john** and for lastName should be **doe**